src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
+  const response = await fetch(`${BASE_URL}/users`);
+
+  return response.json();
+}
```

Comment:
1. Move `getAllUsers` method to separate file - `src/api/users.ts` (leave file `src/api/api.ts` only for api configuration)